Tasks/AzureRmWebAppDeployment/XMLSubstituter.ps1

```diff
@@ -0,0 +1,74 @@
+﻿Param(
+  [Parameter(mandatory=$true)][string]$xmlFilePath,
+  [Parameter(mandatory=$true)][string]$tag,
+  [string]$conditionalAttributeKey,
```

Comment:
inordewr to be consistent with syntax, use [Parameter(mandatory=$false)] and also first specify object type and then whether it is mandatory or not . like [String][Parameter(Mandatory=$true)] $webAppName check checked in task in master branch